src/figuresClasses.ts

```diff
@@ -1,19 +1,94 @@
 export interface Figure {
-
+  shape: string;
```

Comment:
```suggestion
  shape: Shape;
```
Consider creating a separate type or an enum for the shape